src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -357,6 +357,7 @@ public sealed class UpDownCounter<T> : Instrument<T> where T : struct
     }
     public sealed class Histogram<T> : Instrument<T> where T : struct
     {
+        public HistogramAdvice<T>? Advice { get { throw null; } }
```

Comment:
What name do we want to use? https://github.com/open-telemetry/opentelemetry-specification/blob/6b5e9404a3c574c1239230d0753665b4add0b977/CHANGELOG.md?plain=1#L501

Reply:
Advice sounds fine to me offhand. Not sure if you had an alternate suggestion?